src/components/LayerControl.svelte

```diff
@@ -1,9 +1,9 @@
 <script lang="ts">
-	import L from 'leaflet';
```

Comment:
Not sure how these tabs got here. If you are a fan of tabs, let me know so we can be consistent.

Reply:
Lets do four spaces